README.md

```diff
@@ -1,25 +1,17 @@
-# DISCLAIMER:
-
-This library is meant to be used with the [unleash-proxy](https://github.com/Unleash/unleash-proxy). The proxy application layer will sit between your unleash instance and your client applications, and provides performance and security benefits. DO NOT TRY to connect this library directly to the unleash instance, as the datasets follow different formats because the proxy only returns evaluated toggle information.
+⚠️ This library can be used with the [Unleash Proxy](https://github.com/Unleash/unleash-proxy) or with the [Unleash front-end API](https://docs.getunleash.io/reference/front-end-api). It is not compatible with the [server-side Unleash client](https://docs.getunleash.io/reference/sdks#server-side-sdks) API. The proxy application layer will sit between your unleash instance and your client applications, and provides performance and security benefits.
```

Comment:
I'm unsure whether we'll need this or not. It's probably not a bad idea to put it in the readme, but it _will_ mean that it's repeated twice in the generated docs. ... which is possibly still not a bad thing 💁🏼

Reply:
For reference: here's what it'll look like: https://unleash-docs-git-docs-readme-client-v2-unleash-team.vercel.app/reference/sdks/react